Vagrantfile

```diff
@@ -14,17 +14,34 @@ Vagrant.configure(2) do |config|
 
   config.vm.provision "shell", inline: <<-SHELL
     sudo apt-get update
-    sudo apt-get install -y autoconf automake build-essential git libfreetype6-dev libpng12-dev libtool nmap pkg-config python-all-dev python-matplotlib python-nose python-numpy python-pip python-zmq sshpass trace-cmd
+    sudo apt-get install -y autoconf automake build-essential git libfreetype6-dev libpng12-dev libtool nmap pkg-config python-all-dev python-matplotlib python-nose python-numpy python-pip python-zmq sshpass trace-cmd openjdk-7-jdk openjdk-7-jre expect
     sudo pip install ipython[notebook] pandas
     sudo apt-get remove -y w3m
 
     ln -s /vagrant /home/vagrant/lisa
+
+    cd /home/vagrant/lisa
+    if [ ! -d ./tools/android-sdk-linux ]; then
```

Comment:
What about `-e` instead of `-d`?  That way the user can symlink the android-sdk-linux installation that they have installed in the system if that's the case.